staging/src/k8s.io/apiserver/pkg/apis/apiserver/types.go

```diff
@@ -71,20 +71,30 @@ type EgressSelection struct {
 
 // Connection provides the configuration for a single egress selection client.
 type Connection struct {
-	// Type is the type of connection used to connect from client to konnectivity server.
-	// Currently supported values are "http-connect" and "direct".
-	Type string
+	// Protocol is the protocol used to connect from client to the konnectivity server.
+	// Supported values are "http-connect," "grpc", and "direct"
+	Protocol string
```

Comment:
Is it optional? Does it default to http-connect?

nit: missing "." at the end of the sentence, here and a few other places.

Reply:
Can you follow the pattern we used in [PodConditionType](https://github.com/kubernetes/kubernetes/blob/bbc035a66cacc345dd54676e6c4e1f0389dbcb5a/staging/src/k8s.io/api/core/v1/types.go#L2436) for Protocol?